airbyte-integrations/csv/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM airbyte/base-java:dev
+COPY --from=openjdk:14.0.2-slim / /
+
+WORKDIR /airbyte
+# todo (cgardens) - naming.
+ENV INTEGRATION_NAME csv
+
+COPY build/libs/${INTEGRATION_NAME}*.jar lib/${INTEGRATION_NAME}.jar
+
+ENV DESTINATION_CLASS io.airbyte.integrations.csv.CsvDestination
```

Comment:
why pass this as env variables instead of named flags/args to a command? it's easier to follow/more typical of how I've seen dockerfiles used to call a command provided by the docker image rather than set env variables that get picked up somewhere down the line

Reply:
i'd like to stick with this for now. i think this will change pretty soon as i figure out a better way to handle including the jar to the classpath. once i do that we can decide if we want to add as arg.

Reply:
this became moot. don't need to have these env variables / args anymore now that the destination holds the main method.